sigstore/_internal/trustroot.py

```diff
@@ -32,9 +38,12 @@
     TrustedRoot as _TrustedRoot,
 )
 
+from sigstore._internal.keyring import Keyring
 from sigstore._internal.tuf import DEFAULT_TUF_URL, STAGING_TUF_URL, TrustUpdater
 from sigstore.errors import MetadataError
 
+RekorKeyring = NewType("RekorKeyring", Keyring)
```

Comment:
This feels a little out of place 🙂 -- IMO we should either put this back into `rekor/client.py` *or* remove `keyring.py` entirely and inline it into this module; I have a slight preference for the latter (since the Keyring abstraction no longer makes sense outside of a trust root).